navigation.py

```diff
@@ -184,6 +212,7 @@ def wait_for_next_page(driver):
 
 
 def try_login(driver, username, password):
+    """ Tries to log in with given credentials. """
```

Comment:
"Tries to log in with the given credentials. If this fails, no error is thrown and the function returns."